doc/development.md

```diff
@@ -283,3 +284,30 @@ Hosts/ports of containers to access functionality:
  - `localhost` - host, `5010` - port for remote JVM debug
  - `http://localhost:8081` - for calling mod-search REST api. Note that header `x-okapi-url: http://api-mock:8080` should be added to request for apis that take okapi url from headers
  - `localhost:29092` - for kafka interaction. If you are sending messages to kafka from java application with `spring-kafka` then this host shoulb be added to `spring.kafka.bootstrap-servers` property of `application.yml`
+
+### Consortium support for Local environment testing
+Consortium feature is defined automatically at runtime by calling /user-tenants endpoint.
+Consortium feature on module enable is defined by 'centralTenantId' tenant parameter.
+
+Invoke the following
+```shell
+curl --location --request POST 'http://localhost:8081/_/tenant' \
+--header 'Content-Type: application/json' \
+--header 'x-okapi-tenant: consortium' \
```

Comment:
It's enabling central tenant. Let's add also enabling of member tenant(one more post request)